src/components/SettingListItem.jsx

```diff
@@ -30,7 +30,11 @@ const SettingListItem = ({ type, title, explanations }) => {
         )}
       </div>
       <div className="my-auto ml-auto">
-        <ToggleButton checked={settings[type]} onToggle={handleToggle} />
+        <ToggleButton
+          checked={settings[type]}
+          onToggle={handleToggle}
+          disabled={type === "isReturnChannel" && !settings.isAdDetect}
```

Comment:
```suggestion
          disabled={type === SETTING_TYPES.RETURN_CHANNEL && !settings.isAdDetect}
```
"isReturnChannel" 값을 직접 문자열로 사용하는 대신,
이미 정의해두신 상수를 활용하면 유지보수나 오타 방지 측면에서 더 안전할 것 같아요 :)

Reply:
좋은 리뷰 감사드립니다! 해당 사항 반영하겠습니다~